host/3.0/publish.yml

```diff
@@ -108,41 +108,41 @@ stages:
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java8-appservice-deprecated
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java8-appservice-quickstart-deprecated
 
-        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11 $TARGET_REGISTRY/java:$(TargetVersion)-java11
-        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-slim $TARGET_REGISTRY/java:$(TargetVersion)-java11-slim
-        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java11-appservice
-        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java11-appservice-quickstart
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11 $TARGET_REGISTRY/java:$(TargetVersion)-java11-deprecated
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-slim $TARGET_REGISTRY/java:$(TargetVersion)-java11-slim-deprecated
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java11-appservice-deprecated
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java11-appservice-quickstart-deprecated
 
-        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-build $TARGET_REGISTRY/java:$(TargetVersion)-java8-build
-        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-build $TARGET_REGISTRY/java:$(TargetVersion)-java11-build
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-build $TARGET_REGISTRY/java:$(TargetVersion)-java8-build-deprecated
+        docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java11-build $TARGET_REGISTRY/java:$(TargetVersion)-java11-build-deprecated
 
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-slim
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-appservice
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-appservice-quickstart
+        docker push $TARGET_REGISTRY/java:$(TargetVersion)-deprecated
+        docker push $TARGET_REGISTRY/java:$(TargetVersion)-slim-deprecated
+        docker push $TARGET_REGISTRY/java:$(TargetVersion)-appservice-deprecated
+        docker push $TARGET_REGISTRY/java:$(TargetVersion)-appservice-quickstart-deprecated
 
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-slim
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-appservice
-        docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-appservice-quickstart
+        docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-deprecated
+        docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-slim-deprecated
```

Comment:
@CooperLink  - instead of replacing the tags - should this be adding additional tag? 

specifically to ensure the warning added here - ⁠#1005 shows up if anyone runs existing tags ?